packages/kujua-sms/kujua-sms/utils.js

```diff
@@ -508,7 +509,8 @@ exports.addError = function(record, error) {
         .replace('{{fields}}', error.fields && error.fields.join(', '))
         .replace('{{form}}', form);
 
-    record.errors ? record.errors.push(error) : record.errors = [error];
+    if(record.errors) record.errors.push(error);
+    else record.errors = [error];
```

Comment:
I'm not a fan of if/else without brackets and new lines.  But maybe that's just me?  Also thought we used spaces after `if`.  

I actually don't think we need to make these changes/merge this branch because I'm refactoring all this.  We will have some new npm modules like `medic-smsparser` and `medic-api-utils`.  The former won't live in couch but the latter will.  So I will take this jshint work/config and apply it to the latter, but not to this repo.  Does that sounds sensible?  No sense in introducing change here while it's being refactored.  If you want to peer up on the refactor stuff that could be fun too...